specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json

```diff
@@ -222,9 +328,7 @@
             "target": {
               "description": "The target capability models for this relationship.",
               "type": "array",
-              "items": {
-                "type": "string"
-              },
+              "items": {},
```

Comment:
You need to specify something, from Swagger spec:
> The value of items is a schema that describes the type and format of array items.

Reply:
The proper type is `string | CapabilityModel`, which Swagger 2.0 cannot express. The best we could do was fall back to an "any" type like this. The only other alternative I can think of would be to exclude `items` entirely, but I don't know how the various tools would respond to an `array` with missing `items`.

Reply:
I see, I think the best approach would be 
```
"items": {
  "type":"object"
}
```
which autorest will understand as "whatever object", and string being a kind of object.

We're on the borderline here, and I would highly you try to generate code from this PR to check that autorest is happy.

Reply:
I confirmed that AutoRest generated the expected types when given an empty schema - `any[]` in TS and `IList<object>` in C# - so I believe this best reflects what we want. It definitely did not like removing `items`.

Reply:
kept it as "items" {}